src/p2p/net_node.inl

```diff
@@ -1963,6 +1970,52 @@ namespace nodetool
     m_gray_peerlist_housekeeping_interval.do_call(boost::bind(&node_server<t_payload_net_handler>::gray_peerlist_housekeeping, this));
     m_peerlist_store_interval.do_call(boost::bind(&node_server<t_payload_net_handler>::store_config, this));
     m_incoming_connections_interval.do_call(boost::bind(&node_server<t_payload_net_handler>::check_incoming_connections, this));
+    m_dns_blocklist_interval.do_call(boost::bind(&node_server<t_payload_net_handler>::update_dns_blocklist, this));
+    return true;
+  }
+  //-----------------------------------------------------------------------------------
+  template<class t_payload_net_handler>
+  bool node_server<t_payload_net_handler>::update_dns_blocklist()
+  {
+    if (!m_enable_dns_blocklist)
+      return true;
+    if (m_nettype != cryptonote::MAINNET)
+      return true;
+
+    static const std::vector<std::string> dns_urls = {
```

Comment:
`static constexpr const std::initializer_list<const char*>` or `static constexpr const char*[]`. And below would just be `(records, {dns_urls.begin(), dns_urls.end()})` or `(records, {std::begin(dns_urls), std::end(dns_urls)})`.

Reply:
Why ? AFAICT this would create strings/vector every time it runs rather than just once, which runs counter to your usual suggetions, so I might be missing the point.